src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -439,6 +462,22 @@ public virtual JwtSecurityToken CreateToken(string issuer = null, string audienc
             return new JwtSecurityToken(header, payload, rawHeader, rawPayload, rawSignature);
         }
 
+        private IEnumerable<Claim> OutboundTransform(IEnumerable<Claim> claims)
+        {
+            foreach (Claim claim in claims)
+            {
+                string type = claim.Type;
```

Comment:
a nit really, but I would do: type = null so that a string copy is not needed for non-mapped claims.